website/docs/community/resources/slack-rules-of-the-road.md

```diff
@@ -1,67 +1,75 @@
+Speak in your own voice, and join in any or all of the conversations that interest you. Share your expertise as a data professional. Make a meme if you're so inclined. Get in a (friendly) debate. You are not limited to only your company's products and services, and making yourself known as a familiar face outside of commercial contexts is one of the most effective ways of building trust with the community. 
 
-**Do not pitch your products or services in dbt Slack**: this isn't the right place for that. Vendors can add enormous value to the community by being there to answer questions about their products when  questions arise.
+Because unaffiliated community members are able to share links in any channel, the most effective way to have your work reach a wider audience is to create things that are genuinely useful to the community. 
 
-Further, **do not use our Slack community for outbound recruitment for a role**. Recruiters should feel free to post opportunities in the #jobs channel, but should not directly contact members about an opportunity.
 
-We appreciate when vendors and recruiters identify themselves clearly in their Slack username. If you see someone pitching products and services in dbt Slack, or contact you directly about an open role, let us know. We’ll delete the message and remind that person about this rule.
+## Handling violations
 
-## Rule 8: Do not demand attention with @channel and @here, or by tagging individuals
-The @channel and @here keywords in Slack are disabled for everyone except admins. If you make a post containing @channel or @here, nothing will happen. Still, we'll send you a link to this rule to help you better understand how dbt Slack operates.
+The point of these rules is not to find opportunities to punish people, but to ensure the longevity of the community. Participation in this community is a privilege, and we reserve the right to remove people from it. 
 
-Do not tag individuals for in-depth assistance in your questions. If someone feels inclined to answer your question, they will do so. We are a community of volunteers, and we're generally pretty responsive and helpful! If nobody has replied to your question, consider if you've asked a question that helps us understand your problem. If you require in-depth, ongoing assistance, we have a wonderful group of experienced dbt consultants in our ecosystem. You can find a full list [here](https://www.getdbt.com/ecosystem/).
+To report an issue or appeal a judgement, email [community@dbtlabs.com](mailto:community@dbtlabs.com) or use the workflows in #moderation-and-administration on Slack.
 
-## Rule 9: Use threads
-The best way to keep conversations coherent in Slack is to use threads. The dbt Slack community uses threads heavily and if you break this convention, a member of the community will let you know.
+Violations related to our anti-harassment policy will result in immediate removal. Other issues are handled in proportion to their impact, and may include:
```

Comment:
would it be removal from slack or the entire dbt community (forum, repo, etc.)?

```suggestion
Violations related to our anti-harassment policy will result in immediate removal from the dbt Slack community. Other issues are handled in proportion to their impact and may include:
```

Reply:
This is intentionally broad so that it applies across all platforms (e.g. we can't say we'll remove you from Slack if you're not _in_ Slack and are harassing someone at a meetup). 

I wrote it intending "removal from the place where you're being offensive", whatever that might be. But if someone is particularly egregious then I wouldn't rule out an across-the-board ejection, so I'm ok with a bit of ambiguity here